api/descriptions/collections/post.md

```diff
@@ -8,20 +8,30 @@ To invoke this endpoint in the Console UI:
 
 ### cURL Request
 
-Refer to the following example cURL command that creates a new collection named `my-collection` and captures all container images named `ubuntu:18.04`:
+Refer to the following example cURL command that creates a new collection named `my-collection`, specifies a HEX color value of #AD3C21, and captures all container images named `ubuntu:18.04`:
 
 ```bash
-$ curl 'https://<CONSOLE>/api/v<VERSION>/collections' \
+$ curl 'https://<CONSOLE>:8083/api/v<VERSION>/collections' \
   -k \
   -X POST \
   -u <USER> \
   -H 'Content-Type: application/json' \
   -d \
 '{
    "name":"my-collection",
-   "images":["ubuntu:18.04"]
+   "images":["ubuntu:18.04"],
+   "hosts":["*"],
+   "labels":["*"],
+   "containers":["*"],
+   "functions":["*"],
+   "namespaces":["*"],
+   "appIDs":["*"],
+   "accountIDs":["*"],
+   "codeRepos":["*"],
+   "clusters":["*"], 
+   "color":"#AD3C21"
 }'
 ```
-Any resource left unspecified, such as `hosts`, `functions`, or `clusters`, is assigned a wildcard by default.
+**Note:** Resources that are not explicitly specified need to have a `*` wildcard applied to them in order to instantiate as mentioned in the example.
```

Comment:
@ssugandh Let's remove this line. You're already adding a lot of info about this issue at the beginning of the endpoint description. No need to repeat it again here.